docs/camino-messenger/introduction.md

```diff
@@ -214,9 +215,11 @@ Adding additional validation logic in our application code or parsers can help e
 
 In the Camino Messenger Protocol, there is no static data inclusion in any of the Search, Validate and Mint request/response messages to keep these messages as small and as efficient as possible. Static data can be pulled and kept up-to-date at the distributor side for mapping, product selection and descriptive content, but also to enrich search responses where required. Through an external schedule, suppliers can be requested for the list of new, changed and discontinued products on a daily or hourly basis, so that the details of these can be updated with the ProductDetails service in order to avoid the static data cache to lag behind. Front-ends and websites can complete the price and availability received in real-time with static data, descriptions, images, etc from this static data cache to keep the search response messages as small as possible.
 
+Together with the use of the serialized protobuf message format, the removal of static data from the realtime messages, will result to at least 50% less traffic, which has a substantial positive impact on the performance of your applications and traffic/rendering expenses.
+
 ## Versioning and Releases
 
```

Comment:
I would suggest adding here a first line saying, v1 is the latest stable release

Reply:
There must be a section for showing releases,
we can do 
v2  (activity, accomodation)
v1 (all services)
something like that.

Reply:
we could add that in the table at the bottom..

Reply:
yes it's better to ad dthe latest version for each service in the table of services.